src/services/recipient.js

```diff
@@ -381,7 +378,7 @@ export async function getGoalsByActivityRecipient(
       existingGoal.goalNumbers = [...existingGoal.goalNumbers, current.goalNumber];
       existingGoal.objectives = reduceObjectives(current, existingGoal);
       existingGoal.objectiveCount = existingGoal.objectives.length;
-
+      existingGoal.grantNumbers = uniq([...existingGoal.grantNumbers, current.grant.number]);
```

Comment:
Do we need to expand/modify the test for this change?

Reply:
👍  https://github.com/HHS/Head-Start-TTADP/pull/1082/commits/5f03be706fd14d03a2099d28d1e7275a0e7902d8